cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -283,6 +292,7 @@ private synchronized void initProposalExecution(Collection<ExecutionProposal> pr
     setRequestedPartitionMovementConcurrency(requestedPartitionMovementConcurrency);
     setRequestedLeadershipMovementConcurrency(requestedLeadershipMovementConcurrency);
     _uuid = uuid;
+    _executionStoppedByUser.set(false);
```

Comment:
looks redundant here since it is set in `startExecution()`

Reply:
actually the set to false is in
```
    if (!ExecutorUtils.partitionsBeingReassigned(_zkUtils).isEmpty()) {
      _executionTaskManager.clear();
      _uuid = null;
      _executionStoppedByUser.set(false);
      // Note that in case there is an ongoing partition reassignment, we do not unpause metric sampling.
      throw new IllegalStateException("There are ongoing partition reassignments.");
    }
```
i.e. error cleanup. So we still need to set it.

Reply:
I feel a more clear way could be move this ` _executionStoppedByUser.set(false)` out of the `if` statement and remove the one in `initProposalExecution()`